### DISCUSSION TITLE
Anyone free i need help. I have an assignment to submit but i am stuck at an error

### DISCUSSION THREAD

**Discussion by ghostbadfame** [vercel/next.js#71525] [Help]
### Summary

https://github.com/ghostbadfame/SmallBig

Getting these errors:

react-dom.development.js:9126 Uncaught TypeError: Cannot read properties of undefined (reading 'User')

Uncaught Error: There was an error while hydrating. Because the error happened outside of a Suspense boundary, the entire root will switch to client rendering.

Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'User')

**Comment by icyJoseph** [comment]
I guess, 

```ts
export const User = mongoose.models.User || mongoose.model("User", userSchema);
```

`models` doesn't exist in mongoose? Try to debug/log the contents of mongosee.
